### ISSUE TITLE
Filter option default state

### ISSUE THREAD

**Issue by garbagerodin** [gitextensions/gitextensions#6346]
## Current behaviour

Filter option reset after GitExtension rerun. The same with "Branches" filter.

## Expected behaviour

Filter option going through GitExtension rerun.

## Steps to reproduce

1. Filter option change to "Author" or other non default options
2. Close GitExtension
3. Run GitExtension
4. Filter option in default state "commit message and hash"

## Screenshots

![untitled](https://user-images.githubusercontent.com/40666182/53869018-5d417e00-4008-11e9-999a-74bccb0688b1.png)

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

**Comment by vbjay** [comment]
Yep and then we will have issues created saying we aren't showing all the
commits. Filter should not be remembered. We already limit by count. If you
want to filter, then apply whatever filter you want.

**Comment by drewnoakes** [comment]
I agree with @vbjay that persisting across sessions as things currently are will cause confusion that leads to issues being filed. I also understand why you (as someone who understands the feature) want this.

Perhaps the real issue here is that we don't make it obvious enough when a filter is applied.

**Comment by gerhardol** [comment]
> Perhaps the real issue here is that we don't make it obvious enough when a filter is applied.

Agree. Any suggestions?

**Comment by garbagerodin** [comment]
My problem is
1) 95% my filter options is "Author" filter.
2) 2-3 GitExtensions rerun every day due system reboot or GitExtensions hang

So, I must regular click to the funnel for check/change current filter option.

**Comment by vbjay** [comment]
Ctrl f

We might be able to save defined filters and allow user to load when they
would like.


We always start with a filter of commit count. So I am not sure when we
would not be filtering.

**Comment by RussKie** [comment]
> 2-3 GitExtensions rerun every day due system reboot or GitExtensions hang

More details please

**Comment by Otiel** [comment]
The issue reported is to remember the filter type, not the filter. It seems valid to me.
